src/themes/elife/lib/articleType.ts

```diff
@@ -0,0 +1,34 @@
+import { create, prepend } from '../../../util'
+import { getType } from './dataProvider'
+import { articleType, articleData } from './query'
+
+const articleTypes = new Map([
+  ['research-article', 'Research Article'],
+  ['replication-study', 'Replication Study'],
```

Comment:
I need to confirm all the types of article that may have an ERA companion article.

Reply:
Here you go:
- Research Article
- Short Report
- Tools and Resources
- Research Advance
- Registered Report
- Replication Study
- Research Communication

Reply:
Thanks @FAtherden-eLife, I have added all of these types. As new types are added would should extend this list.